colin/core/checks/fmf_check.py

```diff
@@ -0,0 +1,60 @@
+"""
+Module with FMF abstract check class
+"""
+
+import copy
+import logging
+
+from .abstract_check import AbstractCheck
+from ..fmf_extension import ExtendedTree
+from ..ruleset.ruleset import get_checks_path
+
+logger = logging.getLogger(__name__)
+
+
+def receive_fmf_metadata(name, path, object_list=False):
```

Comment:
I wish this function was integrated here: https://github.com/user-cont/colin/blob/master/colin/core/loader.py

so that you don't have to pass the path in such a hacky way (from cli via changing attribute on a class)

Reply:
this is fixed, metadata loaded in loader.py